### PR TITLE
[stable/graylog] Chart improvements

### PR DESCRIPTION
**Pull request by CVirus**
#### What this PR does / why we need it:
- Gives users the ability to prepend the http scheme to the ES URI that was fetched from a k8s secret.
- Changes the default name of the used k8s secrets to avoid potential collision with the main graylog secret.

#### Checklist
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/graylog/templates/_helpers.tpl [hunk 1]
```diff
@@ -78,7 +78,11 @@ Or use chart dependencies with release name
 */}}
 {{- define "graylog.elasticsearch.hosts" -}}
 {{- if .Values.graylog.elasticsearch.uriSecretKey }}
-    {{- printf "${GRAYLOG_ELASTICSEARCH_HOST}" -}}
+    {{- if .Values.graylog.elasticsearch.uriSSL }}
+        {{- printf "https://${GRAYLOG_ELASTICSEARCH_HOST}" -}}
+    {{- else }}
+        {{- printf "http://${GRAYLOG_ELASTICSEARCH_HOST}" -}}
+    {{- end }}
```
Comment (KongZ) [inline stable/graylog/templates/_helpers.tpl:85]: Why not just let user add `https` in `.Values.graylog.elasticsearch.uriSecretKey` value?
It's less complicate than having another `uriSSL` prefix.

Comment (CVirus) [reply stable/graylog/templates/_helpers.tpl:85]: The use case this whole uriSecretKey, uriSecretName and uriSSL features target is the [aws-servicebroker](https://github.com/awslabs/aws-servicebroker). Unfortunately, the broker doesn't provide a way to add `http` or `https` to the secret it generates from the servicebinding.

Comment (KongZ) [reply stable/graylog/templates/_helpers.tpl:85]: it seems that aws-servicebroker is a kind of service that mess up others

**Pull request merged**